index.html

```diff
@@ -15,66 +15,73 @@
-      <h2 class='tech-title'>NodeJS</h2>
-      <p class='tech-release'>Release:2009</p>
-      <p class='tech-desc'>Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.</p>
-    </div>
-    <div class='tech'>
-      <img class='tech-img' src='Assets/aspnet_icon.png'>
-      <h2 class='tech-title'>ASP.NET Core</h2>
-      <p class='tech-release'>Release:2016</p>
-      <p class='tech-desc'>ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and te cross-platform .NET Core.</p>
-    </div>
-    <div class='tech'>
-      <img class='tech-img' src='Assets/django_icon.svg'>
-      <h2 class='tech-title'>Django</h2>
-      <p class='tech-release'>Release:2005</p>
-      <p class='tech-desc'>Django (/ˈdʒæŋɡoʊ/ JANG-goh) is a free open-source web framework, written in Pyhton, which follows the model-view-template (MVT) architectural pattern.</p>
+    <h1 class='title'>Back-End Tools</h1>
+    <div class='tech-list'>
+      <div class='tech'>
+        <img class='tech-img' src='Assets/rails_icon.png'>
+        <h2 class='tech-title'>Ruby on Rails</h2>
+        <p class='tech-release'>Release:2005</p>
+        <p class='tech-desc'>Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</p>
+      </div>
+      <div class='tech'>
+        <img class='tech-img' src='Assets/express_icon.svg'>
+        <h2 class='tech-title'>NodeJS</h2>
+        <p class='tech-release'>Release:2009</p>
+        <p class='tech-desc'>Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.</p>
+      </div>
+      <div class='tech'>
+        <img class='tech-img' src='Assets/aspnet_icon.png'>
+        <h2 class='tech-title'>ASP.NET Core</h2>
+        <p class='tech-release'>Release:2016</p>
+        <p class='tech-desc'>ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and te cross-platform .NET Core.</p>
+      </div>
+      <div class='tech'>
+        <img class='tech-img' src='Assets/django_icon.svg'>
+        <h2 class='tech-title'>Django</h2>
+        <p class='tech-release'>Release:2005</p>
+        <p class='tech-desc'>Django (/ˈdʒæŋɡoʊ/ JANG-goh) is a free open-source web framework, written in Pyhton, which follows the model-view-template (MVT) architectural pattern.</p>
+      </div>
     </div>
   </div>
+  <footer class='footer'>
+    <h1 class='footer-subtitle'>WE ARE GAME CHANGERS</h1>
+    <img class='footer-logo' src='Assets/wolox_logo_white.svg'>
+    <h2 class='footer-copyright'>Copyright 2018. Wolox. All rights reserved.</h2>
```

Comment:
I'd use a `span` or a `p` here instead of `h2` since it's just a small text that's not really important